projects/picker/src/lib/date-time/calendar.component.html

```diff
@@ -1,108 +1,165 @@
+                                s9.233-1.811,12.847-5.428L286.935,95.074c3.613-3.617,5.427-7.898,5.427-12.847C292.362,77.279,290.548,72.998,286.935,69.377z"
+                            />
                         </g>
                     </svg>
                     <!-- </editor-fold> -->
                 </span>
             </span>
         </button>
     </div>
-    <button class="owl-dt-control owl-dt-control-button owl-dt-control-arrow-button"
-            type="button" tabindex="0"
-            [style.visibility]="showControlArrows? 'visible': 'hidden'"
-            [disabled]="!nextButtonEnabled()"
-            [attr.aria-label]="nextButtonLabel"
-            (click)="nextClicked()">
-        <span class="owl-dt-control-content owl-dt-control-button-content" tabindex="-1">
+    <button
+        class="owl-dt-control owl-dt-control-button owl-dt-control-arrow-button"
+        type="button"
+        tabindex="0"
+        [style.visibility]="showControlArrows ? 'visible' : 'hidden'"
+        [disabled]="!nextButtonEnabled()"
+        [attr.aria-label]="nextButtonLabel"
+        (click)="nextClicked()"
+    >
+        <span
+            class="owl-dt-control-content owl-dt-control-button-content"
+            tabindex="-1"
+        >
             <!-- <editor-fold desc="SVG Arrow Right"> -->
-        <svg version="1.1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" x="0px" y="0px"
-                 viewBox="0 0 250.738 250.738" style="enable-background:new 0 0 250.738 250.738;" xml:space="preserve">
-                <path style="fill-rule:evenodd;clip-rule:evenodd;" d="M191.75,111.689L84.766,5.291c-7.1-7.055-18.613-7.055-25.713,0
+            <svg
+                version="1.1"
+                xmlns="http://www.w3.org/2000/svg"
+                xmlns:xlink="http://www.w3.org/1999/xlink"
+                x="0px"
+                y="0px"
+                viewBox="0 0 250.738 250.738"
+                style="enable-background: new 0 0 250.738 250.738"
+                xml:space="preserve"
+            >
+                <path
+                    style="fill-rule: evenodd; clip-rule: evenodd"
+                    d="M191.75,111.689L84.766,5.291c-7.1-7.055-18.613-7.055-25.713,0
                     c-7.101,7.054-7.101,18.49,0,25.544l95.053,94.534l-95.053,94.533c-7.101,7.054-7.101,18.491,0,25.545
                     c7.1,7.054,18.613,7.054,25.713,0L191.75,139.05c3.784-3.759,5.487-8.759,5.238-13.681
-                    C197.237,120.447,195.534,115.448,191.75,111.689z"/>
+                    C197.237,120.447,195.534,115.448,191.75,111.689z"
+                />
             </svg>
             <!-- </editor-fold> -->
         </span>
     </button>
 </div>
-<div class="owl-dt-calendar-main" cdkMonitorSubtreeFocus [ngSwitch]="currentView" tabindex="-1">
+<div class="owl-dt-calendar-main" cdkMonitorSubtreeFocus tabindex="-1">
+    @switch (currentView) { @case (DateView.MONTH) {
```

Comment:
was this formatted with a recent prettier version?
This seems wrong, as if it was a version where prettier did not understand the new control flow.